### ISSUE TITLE
SLES 11 SP4 fails with pseudo SELinux

### ISSUE THREAD

**Issue by lazyfrosch** [Icinga/puppet-icinga_build#33]
In our CoreOS environment, there is some selinux, but not enabled, and not enabled inside the build containers.

It still fails on useradd / groupadd. Most likely due to the scripts seeing `/selinux` as mount point.

```
Installation of icinga2-common-2.6.3.296.g60b45ae-0.2017.05.26+1.icinga failed:
(with --nodeps --force) Error: Subprocess failed. Error: RPM failed: Cannot set attribute security.selinux for `/etc/group.tmpXonrvF': Operation not supported
Cannot set attribute security.selinux for `/etc/group.tmp2DgQyI': Operation not supported
useradd: Unknown group `icinga'.
error: %pre(icinga2-common-2.6.3.296.g60b45ae-0.2017.05.26+1.icinga.x86_64) scriptlet failed, exit status 6
error:   install: %pre scriptlet failed (2), skipping icinga2-common-2.6.3.296.g60b45ae-0.2017.05.26+1.icinga
```

**Comment by lazyfrosch** [comment]
We will go a workaround for now:

```bash
    # Workaround for SLES 11 SP4
    if [[ "$dist" = 11.* ]]; then
      # which does not like to work with a SELinux host...
      getent group icinga >/dev/null || echo 'icinga:!:998:' >> /etc/group
      getent group icingacmd >/dev/null || echo 'icingacmd:!:997:icinga' >> /etc/group
      getent group icingaweb2 >/dev/null || echo 'icingaweb2:!:996:' >> /etc/group
      if ! getent passwd icinga >/dev/null; then
        echo 'icinga:x:998:998:icinga:/var/spool/icinga2:/sbin/nologin' >> /etc/passwd
        echo 'icinga:!:17310:0:99999:7:::' >> /etc/shadow
      fi
    fi
```

In the rpm_test script

**Comment by lazyfrosch** [comment]
Fixed by 07412ea800617538fe1e9eb6f75e57921f13b718
